Sources/ComposableArchitectureTestSupport/TestStore.swift

```diff
@@ -166,8 +166,8 @@ extension TestStore where LocalState: Equatable {
 
     var cancellables: [AnyCancellable] = []
 
-    func runReducer(state: inout State, action: Action) {
-      let effect = self.reducer.callAsFunction(&state, action, self.environment)
+    func runReducer(action: Action) {
+      let effect = self.reducer.callAsFunction(self.state, action, self.environment)
```

Comment:
I believe this needs to be:

```suggestion
      let effect = self.reducer.callAsFunction(&self.state, action, self.environment)
```

Reply:
oh crap, thanks.